test/hoss/HOSS_Regression.ipynb

```diff
@@ -404,7 +379,7 @@
     "\n",
     "\n",
     "    submit_and_download(harmony_client, hoss_temporal_request, hoss_temporal_file_name)\n",
-    "    assert exists(all_no_bbox_file_name), 'Unsuccessful HOSS temporal request.'\n",
+    "    assert exists(hoss_temporal_file_name), 'Unsuccessful HOSS temporal request.'\n",
```

Comment:
I'm glad these updates were necessary, as this was previously checking the wrong file existed for this test!